pkg/capabilities/consensus/ocr3/benchmark_test.go

```diff
@@ -0,0 +1,487 @@
+package ocr3_test
+
+import (
+	"context"
+	"fmt"
+	"runtime"
+	"testing"
+	"time"
+
+	"github.com/shopspring/decimal"
+	ocrcommon "github.com/smartcontractkit/libocr/commontypes"
+	"github.com/smartcontractkit/libocr/offchainreporting2/types"
+	"github.com/smartcontractkit/libocr/offchainreporting2plus/ocr3types"
+	"github.com/stretchr/testify/require"
+	"go.uber.org/zap/zapcore"
+	"google.golang.org/protobuf/proto"
+	"google.golang.org/protobuf/types/known/timestamppb"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/capabilities/consensus/ocr3"
+	"github.com/smartcontractkit/chainlink-common/pkg/capabilities/consensus/ocr3/datafeeds"
+	"github.com/smartcontractkit/chainlink-common/pkg/capabilities/consensus/ocr3/requests"
+	pbtypes "github.com/smartcontractkit/chainlink-common/pkg/capabilities/consensus/ocr3/types"
+	"github.com/smartcontractkit/chainlink-common/pkg/capabilities/datastreams"
+	"github.com/smartcontractkit/chainlink-common/pkg/logger"
+	"github.com/smartcontractkit/chainlink-common/pkg/values"
+)
+
+// mockCapability implements CapabilityIface for testing
+type mockCapability struct {
+	aggregators map[string]pbtypes.Aggregator
+}
+
+func (m *mockCapability) GetAggregator(workflowID string) (pbtypes.Aggregator, error) {
+	return m.aggregators[workflowID], nil
+}
+
+func (m *mockCapability) GetEncoderByWorkflowID(workflowID string) (pbtypes.Encoder, error) {
+	return nil, nil // Not used in benchmark
```

Comment:
Very minor: Panicking here would avoid any tests added in future accidentally using these functions and needing debugged before noticing they aren't implemented

```suggestion
	panic("Not implemented") // Not used in benchmark
```